packages/flutter/lib/src/material/grid_tile_bar.dart

```diff
@@ -64,13 +64,9 @@ class GridTileBar extends StatelessWidget {
     if (backgroundColor != null)
       decoration = new BoxDecoration(backgroundColor: backgroundColor);
 
-    EdgeInsets padding;
-    if (leading != null && trailing != null)
-      padding = const EdgeInsets.symmetric(vertical: 16.0, horizontal: 8.0);
-    else if (leading != null)
-      padding = const EdgeInsets.only(left: 8.0, right: 16.0, top: 16.0, bottom: 16.0);
-    else // trailing != null || (leading == null && trailing == null)
-      padding = const EdgeInsets.only(left: 16.0, right: 8.0, top: 16.0, bottom: 16.0);
+    EdgeInsets padding = (leading != null)
+      ? const EdgeInsets.only(left: 8.0, right: 16.0)
+      : const EdgeInsets.only(left: 16.0, right: 8.0);
```

Comment:
Why are there only two cases now?  I Would have expected the `trailing == null` and `trailing != null` cases to be treated differently.